### PR TITLE
Add integration doc for Leviosa_Shades

### PR DESCRIPTION
**Pull request by greg-ha-1990**
## Proposed change

Add integration doc for Leviosa_Shades

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/103683
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4755
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/leviosa_shades.markdown [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: Leviosa Motor Blinds Zone hub
+description: Instructions on how to integrate the Leviosa Zone hub into Home Assistant.
+ha_category:
+  - Cover
+ha_iot_class: Local Push
+ha_release: 2021.4
+ha_domain: leviosa_shades
+ha_codeowners:
+  - '@altersis'
```
Comment (SeraphicRav) [inline source/_integrations/leviosa_shades.markdown]: I think the code owner should be @greg-ha-1990

File: source/_integrations/leviosa_shades.markdown [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+title: Leviosa Motor Blinds Zone hub
+description: Instructions on how to integrate the Leviosa Zone hub into Home Assistant.
+ha_category:
+  - Cover
+ha_iot_class: Local Push
+ha_release: 2021.4
```
Comment (SeraphicRav) [inline source/_integrations/leviosa_shades.markdown]: Release should be 2023.11 if everything goes well

File: source/_integrations/leviosa_shades.markdown [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+When you have multiple Leviosa Zones, you'll need to know which IP address is associated with each Leviosa Zone hub. The easiest way to do this is to:
+
+- Open the Leviosa App.
+- Press and hold on any shade group name. After 1 second, a new screen will appear.
+- Select the ‘Name’ tab at the top.
+- The screen will show the IP address of the Leviosa Zone that shade group is on.
+
+## Services to move your blinds to intermediate positions
+
+When you want your blinds to go to intermediate positions defined by you, set these intermediate positions with the [Leviosa remote controller](https://cdn.shopify.com/s/files/1/1346/0347/files/Leviosa_Shades_Programming_instructions.pdf?). These intermediate positions are stored in each Leviosa shade motor, and two services in Home Assistant tell shades in each group to move there. If you do not program intermediate points, the shade willfully travels to the upper or lower limit when these services are used. Once the intermediate positions are set, you can:
```
Comment (SeraphicRav) [inline source/_integrations/leviosa_shades.markdown]:
```suggestion
When you want your blinds to go to intermediate positions defined by you, set these intermediate positions with the [Leviosa remote controller](https://cdn.shopify.com/s/files/1/1346/0347/files/Leviosa_Shades_Programming_instructions.pdf?). These intermediate positions are stored in each Leviosa shade motor, and two services in Home Assistant tell shades in each group to move there. If you do not program intermediate points, the shade will instead travel to the upper or lower limit when these services are used. Once the intermediate positions are set, you can:
```

File: source/_integrations/leviosa_shades.markdown (hunk 1 above)
Comment (greg-ha-1990) [reply source/_integrations/leviosa_shades.markdown]: It was my mistake, I have submitted an update

File: source/_integrations/leviosa_shades.markdown (hunk 2 above)
Comment (greg-ha-1990) [reply source/_integrations/leviosa_shades.markdown]: I have submitted an update

File: source/_integrations/leviosa_shades.markdown (hunk 3 above)
Comment (greg-ha-1990) [reply source/_integrations/leviosa_shades.markdown]: I have submitted an update

**Review by SeraphicRav — APPROVED** [review]
Looks good to me.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**